lib/data_update_scripts/20200217215802_index_listings_to_elasticsearch.rb

```diff
@@ -1,9 +1,9 @@
 module DataUpdateScripts
```

Comment:
I don't think we can change the name of this, as the data update script machinery stores the name of the file. This will either result in a situation of "there are missing files to be run" or re-run this script. Not sure which of the two